addons/spreadsheet/static/src/list/plugins/list_ui_plugin.js

```diff
@@ -236,6 +242,27 @@ export class ListUIPlugin extends spreadsheet.UIPlugin {
         this.dataSources.get(dataSourceId).increaseMaxPosition(position);
     }
 
+    _getUnusedLists() {
+        if (this.unusedLists !== undefined) {
+            return this.unusedLists;
+        }
+        const unusedLists = new Set(this.getters.getListIds());
+        for (const sheetId of this.getters.getSheetIds()) {
+            for (const cellId in this.getters.getCells(sheetId)) {
+                const position = this.getters.getCellPosition(cellId);
+                const listId = this.getListIdFromPosition(position);
+                if (listId) {
+                    unusedLists.delete(listId);
+                    if (!unusedLists.size) {
```

Comment:
Here we don't create this.unusedLists as an empty list, is it a mistake or an intended behavior ?

Reply:
Mistake indeed :)